common/persistence/src/main/resources/db/migration/V9__alterMetadataTables.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE key_submission_metadata_with_client_metadata
+  ADD submitted_with_check_ins BOOLEAN;
+ALTER TABLE key_submission_metadata_with_client_metadata ALTER COLUMN submitted_with_check_ins SET DEFAULT FALSE;
+
+ALTER TABLE key_submission_metadata_with_user_metadata
+  ADD submitted_after_rapid_antigen_test BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
@mlenkeit - awaiting more information, how to map protobuf (two-state-boolean) into postgres (tri-state-boolean)